### PR TITLE
Ar 1050 add new collection type public events

### PR DESCRIPTION
**Pull request by davgur**

### CODE REVIEW

File: public/locales/en/common.json [hunk 1]
```diff
@@ -134,13 +134,16 @@
       "LESSON_PART": "Daily Lessons",
       "DAILY_LESSON": "Daily Lessons",
       "VIRTUAL_LESSON": "Webinars",
+      "VIRTUAL_LESSONS": "Webinars",
```
Comment (gshilin) [inline public/locales/en/common.json:137]: What's difference between VIRTUAL_LESSON and VIRTUAL_LESSONs?

File: public/locales/en/common.json [hunk 2]
```diff
@@ -134,13 +134,16 @@
...
       "LECTURE": "Lectures",
       "WOMEN_LESSON": "Women Lessons",
       "CHILDREN_LESSON": "Children Lessons",
       "LESSONS_SERIES": "Study Series",
       "LECTURE_SERIES": "Lecture Series",
       "CLIP": "Clips",
+      "CLIPS": "Clips",
```
Comment (gshilin) [inline public/locales/en/common.json:144]: What's difference between CLIP and CLIPS?

File: public/locales/en/common.json [hunk 3]
```diff
@@ -149,7 +152,8 @@
       "LIKUTIM": "Assorted Excerpts",
       "SOURCE": "Kabbalah Library",
       "PICNIC": "Picnic",
-      "R_TWEET": "Tweets"
+      "R_TWEET": "Tweets",
+      "PUBLIC_EVENTS": "Public Events"
```
Comment (gshilin) [inline public/locales/en/common.json:156]: Would be nice to use google translator and to translate it to different languages

File: src/components/Sections/Programs/CollectionsByCtBtn.js [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+  filtersGetFilterByNameSelector,
+  settingsGetLeftRightByDirSelector
+} from '../../../redux/selectors';
+import { useTranslation } from 'react-i18next';
+import CollectionsByCtModal from '../../FiltersAside/CollectionsByCt/CollectionsByCtModal';
+
+const cCtByUnitCt = {
+  [CT_VIDEO_PROGRAM_CHAPTER]: CT_VIDEO_PROGRAM,
+  [CT_CLIP]                 : CT_CLIPS,
+  [CT_VIDEO_PROGRAM]        : CT_VIDEO_PROGRAM_CHAPTER,
```
Comment (gshilin) [inline src/components/Sections/Programs/CollectionsByCtBtn.js:25]: Lines 23 and 25 - are you kidding?

File: public/locales/en/common.json (hunk 1 above)
Comment (davgur) [reply public/locales/en/common.json:137]: VIRTUAL_LESSONs is collection content type that have same title on filters

File: public/locales/en/common.json (hunk 2 above)
Comment (davgur) [reply public/locales/en/common.json:144]: same as VIRTUAL_LESSONs

File: public/locales/en/common.json (hunk 3 above)
Comment (davgur) [reply public/locales/en/common.json:156]: we do it for 4 main languages

File: src/components/Sections/Programs/CollectionsByCtBtn.js (hunk 4 above)
Comment (davgur) [reply src/components/Sections/Programs/CollectionsByCtBtn.js:25]: its need for same title of filter

**Pull request merged**